compiler/ast.nim

```diff
@@ -223,6 +223,7 @@ type
     nkBreakState,         # special break statement for easier code generation
     nkFuncDef,            # a func
     nkTupleConstr         # a tuple constructor
+    nkExportDoc,          # a node for export marker and doc comment like ``a* ## comment``
```

Comment:
what if we have a doc comment without a `*`?
I would still want tooling to be able to access doc comment for these.
(or maybe I misunderstood what is meant here?)

Reply:
you get a nkExportDoc if you have either a doccomment, an export marker or both.

Reply:
```suggestion
    nkExportDoc,          # a node for export marker,  doc comment, or both, eg ``a* ## comment``
```